src/content/api/module-methods.md

```diff
@@ -72,7 +73,9 @@ if ( module.hot ) {
 
 W> This feature relies on [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) internally. If you use `import()` with older browsers, remember to shim `Promise` using a polyfill such as [es6-promise](https://github.com/stefanpenner/es6-promise) or [promise-polyfill](https://github.com/taylorhakes/promise-polyfill).
 
-The spec for `import` doesn't allow control over the chunk's name or other properties as "chunks" are only a concept within webpack. Luckily webpack allows some special parameters via comments so as to not break the spec:
+## Magic Comments
+
+The spec for `import` doesn't allow control over the chunk's name or other properties as "chunks" are only a concept within webpack. Luckily webpack allows some special parameters via comments so as to not break the spec. It really is as simple as just adding these comments in order to make these features work:
```

Comment:
I don't understand this part:
> The spec for `import` doesn't allow control over the chunk's name or other properties as "chunks" are only a concept within webpack.

Do you mean:
"The spec for `import` doesn't allow control over the chunk's name or other properties as "chunks". Chunks is only a webpack concept." ?

Reply:
Also, please remove or rephrase:
> It really is as simple as just adding these comments in order to make these features work:

What is simple for one person it is not for everybody.

Reply:
It was there already, means there is no "chunks" in spec, but webpack had come up with it :) Though i dont really like the sentence that was added, it does not give any information "simple comments to make features work" where we had already stated that webpack: "allows parameters via comments". literally repeating ourselves with other manner

Reply:
ok have something that might work better